m4_sessions/1801-inning/group_2.md

```diff
@@ -1,12 +1,23 @@
 Group 2
 
-Group Members:
+Group Members: Avoiding Burnout
 
-Name of Session:
+Name of Session: Julie D, Rob M, Pat, Miguel
 
 Session Description (Why should students come to this session):
+• burnout very common problem at Turing and in real world. 
+• Can be hard to avoid and come back from
 
 Session Outcomes (What will students learn/leave with?):
+• Students will learn how to pace themselves
+• gain perspective
+• work-life balance
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+1. Intro/Ice Breakers
+2. Discussion on what cause burnout
+  a) imposter syndrom
+  b) failure
+3. TedTalk
```

Comment:
Do you have a slack channel to share resources?  If you do, please invite me.  There are a ton of potential talks so narrowing it down will be important.  Try for between 10-15 minutes of a video.  Longer is often too long for a session.